tendrl/node_agent/discovery_modules/platform/manager.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import importlib
+import inspect
+from tendrl.node_agent.discovery_modules.platform import base
+
+
+LOG = logging.getLogger(__name__)
+
+
+class PlatformManager(object):
```

Comment:
We wont be able to use common detection manager as we talked about. The plugins use meta class PluginMount and that maintains a list of plugins loaded as instance variable. If we maintain all the platform, sds etc plugins in same list, while picking and executing in node agent manager would be problematic.
We would end up writing different managers for different detection modules. Lets see, anyway I would write one for sds detection as well may be.